Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
Beside logging every website I visit and making it accessible to random chunks of the government on the flimsiest of reasons I'm fortunate to live in a fairly free country, so I can't be certain of this, but there's a potential these caveats remain a useful nudge towards documentation in places where the "reasonable defaults" applied if you don't edit the `torrc` and remove the `.sample` from the filename are not entirely reasonable.